### PR TITLE
[doc] Add README instructions to build using Docker CLI

### PR DESCRIPTION
**Pull request by saimidu**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
This commit adds instructions on building DLC dockerfiles into images using the Docker CLI, rather than python scripts, to offer users more granular control over the build process.

## Label Checklist
- [x] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,30 @@ folder structure as per above and modify the buildspec.yml file to specify the v
                        --py_versions py3
     ```
 
+### Building your image using Docker CLI
+
+To build individual dockerfiles with a much more granular look at layer builds, you can use Docker CLI commands to build DLC
+images by following these steps:
```
Comment (arjkesh) [inline README.md:111]: Nit: we're building images, not dockerfiles here - also there are other use cases than getting a granular look at layers. I'd change this sentence to the following:

```suggestion
If you want to build your docker image directly with Docker CLI instead of using our build scripts, follow these steps below:
```

File: README.md [hunk 2]
```diff
@@ -105,6 +105,30 @@ folder structure as per above and modify the buildspec.yml file to specify the v
                        --py_versions py3
     ```
 
+### Building your image using Docker CLI
```
Comment (arjkesh) [inline README.md:108]: Note: if we add this to our README, we'll have to support questions/issues associated with building images directly through docker CLI. It will be worth it to call out (in a note) that this is not a regularly tested operation

Comment (saimidu) [reply README.md:108]: Done. Please take a look at the "Note:" section.

**Review by arjkesh — APPROVED** [review]

**Review by access2rohit — APPROVED** [review]
LGTM! Thanks for adding this
